assets/js/components/InstanceOverview/InstanceOverview.test.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { screen, waitFor, act } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { databaseInstanceFactory } from '@lib/test-utils/factories';
+import { renderWithRouter } from '@lib/test-utils';
+import { DATABASE_TYPE, APPLICATION_TYPE } from '@lib/model';
+import { faker } from '@faker-js/faker';
+import InstanceOverview from './InstanceOverview';
+
+describe('InstanceOverview', () => {
+  it('renders HealthIcon for a registered instance', () => {
```

Comment:
we start all the tests with a `should` statement by convention (i didn't find any that we don't)